tempel.el

```diff
@@ -108,7 +108,7 @@ must return a list of templates which apply to the buffer or context."
 (defvar tempel--file-templates nil
   "Templates loaded from the `tempel-file'.")
```

Comment:
Call this `tempel--path-templates`. Grep for `tempel-file`, it shouldn't be mentioned anywhere anymore.